### ISSUE TITLE
Arrow Icons cannot be read by screen reader

### ISSUE THREAD

**Issue by mdctenga** [grommet/hpe-digitaltoolkit#127]
## Objective

When a screenreader is focused on the arrow icon, it should say "arrow icon"

### Steps to Reproduce

1. Navigate to [http://hpe-toolkit.herokuapp.com/docs/hpe/examples](http://hpe-toolkit.herokuapp.com/docs/hpe/examples) with a screen reader on.
2. Move the screen reader's focus to an arrow icon. Notice that the screen reader says "undefined-icon"
![screen shot 2016-09-06 at 5 33 30 pm](https://cloud.githubusercontent.com/assets/8364525/18298976/9fc1eff4-7458-11e6-9310-778b4599a362.png)

**Comment by mdctenga** [comment]
I think it just needs an alt text.

**Comment by codeswan** [comment]
This is not a Digital Toolkit bug.  That icon is the default icon for the `Anchor` component as set in grommet-core, not something that is set in Digital Toolkit.

**Comment by davidcalhoun** [comment]
[arrow-screenreader.mp4.zip](https://github.com/grommet/hpe-digitaltoolkit/files/543073/arrow-screenreader.mp4.zip)

Seems like it works now?  OSX screenreader announces "link next, learn more" from the `aria-label` (see attachment)

<img width="637" alt="screen shot 2016-10-20 at 3 31 12 pm" src="https://cloud.githubusercontent.com/assets/120596/19580316/57227382-96da-11e6-81c7-0ed73902c056.png">
